test/mitmproxy/test_flow_export.py

```diff
@@ -1,361 +1,361 @@
-from textwrap import dedent

```

Comment:
Your branch looks a bit messy with a lot of diff.
Could you please try to rebase to current master and keep the file modes intact (0644 vs. 0755).
Ideally Github should only list very few files in the "Files changed" tab - and not 106!